modules/tile-converter/src/i3s-converter/helpers/geometry-converter.ts

```diff
@@ -69,7 +69,8 @@ export default async function convertB3dmToI3sGeometry(
   attributeStorageInfo: AttributeStorageInfo[] | undefined,
   draco: boolean,
   generateBoundingVolumes: boolean,
-  geoidHeightModel: Geoid
+  geoidHeightModel: Geoid,
+  workerSource: {[key: string]: string}
```

Comment:
I load all required workers source code in the initial stage of the conversion. This way source code of a worker is loaded once and is reused for every `WorkerThread` in the pool

Reply:
Also there is and error in nodejs
`(node:54695) UnhandledPromiseRejectionWarning: TypeError [ERR_WORKER_PATH]: The worker script or module filename must be an absolute path or a relative path starting with './' or '../'. Received "https://unpkg.com/@loaders.gl/draco@3.2.0-alpha.1/dist/draco-writer-nodejs-worker.js"`
It works with source instead of URL so it is loaded out of `worker-utils` scope.